model/task_lifecycle_test.go

```diff
@@ -6316,6 +6343,14 @@ func TestUpdateUnblockedDependencies(t *testing.T) {
 	dbTask5, err := task.FindOneId(tasks[5].Id)
 	assert.NoError(err)
 	assert.True(dbTask5.DependsOn[0].Unattainable)
+
+	// Restarting a dependent task should unblock builds
```

Comment:
nit: this isn't really testing restarting so much as unit testing one specific piece of functionality that's runs as part of restarting. I would just phrase this as something like `Unblocking a dependent task should unblock its build` so it's not confusing.

Reply:
done